backend/controllers/createAuthUser.js

```diff
@@ -46,6 +46,7 @@ const createAuthUserFactory = (
       try {
         const userId = await createAuthUser(res.locals, user)
 
+        req.session.searchUrl = searchUrl
```

Comment:
There was an existing bug in creation of auth users - the user details page would end up showing a DPS search link.  This corrects this.